crates/orderbook/src/fee.rs

```diff
@@ -150,6 +159,9 @@ impl From<u32> for FeeParameters {
 #[cfg_attr(test, mockall::automock)]
 #[async_trait::async_trait]
 pub trait MinFeeCalculating: Send + Sync {
+    // TODO: User should be mandatory. It is not because some old routes like /api/v1/fee do not
+    // know the user. When we remove those apis we can remove the Optional.
+
```

Comment:
Instead, can't the `api/v1/fee` just use `H160::default()`? Note that `address(0)` is guaranteed to never have a balance:

https://github.com/gnosis/cow-token/blob/6cc9c8ba02116869b319a705f933bcad99633529/src/contracts/vendored/mixins/ERC20.sol#L272-L273

Reply:
I was testing with weth where the 0 address actually has a lot of balance so I thought optional was better for now. Since the cow token does not allow this agree we can use it instead of None.

Reply:
Done